mindee/client.py

```diff
@@ -229,6 +240,10 @@ def _predict_async(
 
         :param doc_config: Configuration of the document.
         """
+        if self.input_doc is None:
+            raise RuntimeError(
+                "The '_predict_async' class method requires an input document."
```

Comment:
Maybe use `TypeError` instead?

Because `None` is of type `NoneType`, which is of a different type than `XxxInputSource`...

https://docs.python.org/3/library/exceptions.html#TypeError